networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json

```
@@ -0,0 +1,15 @@
+{
+    "CMCslug": "",
+    "decimals": 18,
+    "mldId": "meld-mape-0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "orgName": "MAPE",
+    "orgUrl": "",
+    "tokenAddress": "0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "tokenColorTheme": 1,
+    "tokenIcon": "/static/tokens/mape.svg",
+    "tokenName": "Mape",
```

Based on explorer data:
```suggestion
    "tokenName": "mape",
```